.github/workflows/ci.yml

```diff
@@ -43,13 +43,13 @@ jobs:
         with:
           token: ${{ secrets.codecov_token }}
 
-
   terraform-plan:
     name: Terraform fmt and plan
     runs-on: ubuntu-latest
     env:
-      tf_version: '0.12.24'
+      tf_version: '0.12.26'
       codecov_token: ${{ secrets.codecov_token }}
+      TF_IN_AUTOMATION: 'true'
```

Comment:
See https://learn.hashicorp.com/terraform/development/running-terraform-in-automation#controlling-terraform-output-in-automation for info about this variable.